src/utils/selectors/QACert/assert.js

```diff
@@ -285,12 +285,11 @@ export const saveDataSwitch = (userInput, name, location, id, extraIdsArr) => {
     case flowRataRun:
       return qaApi
         .updateFlowRunData(
-          location,
-          id,
           extraIdsArr[0],
           extraIdsArr[1],
           extraIdsArr[2],
           extraIdsArr[3],
+          id,
```

Comment:
This update is related to editing flow rata run. is this with in the scope of ticket being worked on? I'm thinking it may break edit functionality for flow run. If not, looks good.

Reply:
This was related to an issue with editing flow rata run not saving data brought up by Moses.